odl/util/npy_compat.py

```diff
@@ -6,72 +6,14 @@
 # v. 2.0. If a copy of the MPL was not distributed with this file, You can
 # obtain one at https://mozilla.org/MPL/2.0/.
 
-"""Numpy functions not available in the minimal required version."""
+"""Numpy functions not available in the minimal required version.
```

Comment:
Well it can easily be filled again with stuff, e.g. when there's a nice new function in NumPy that simplifies some code, and we want to backport it to earlier versions.
So to avoid deleting and re-introducing the file again, I'd suggest to keep it around.